site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace app\controllers;
+
+use app\libraries\Core;
+use app\libraries\response\Response;
+use app\libraries\response\WebResponse;
+use app\libraries\response\JsonResponse;
+use app\libraries\response\RedirectResponse;
+use app\models\Notification;
+use Symfony\Component\Routing\Annotation\Route;
+use app\models\OfficeHoursQueueStudent;
+use app\site\libraries\database\DatabaseQueries;
+use app\libraries\routers\AccessControl;
+
+/**
+ * Class OfficeHourQueueController
+ *
+ */
+class OfficeHourQueueController extends AbstractController {
```

Comment:
You've named your file with an **s** at the end of `Hour` and your class without it, that's why phpstan is complaining.

Reply:
Good catch, thanks